demo/dashboard/client/src/lib/getIcon.js

```diff
@@ -0,0 +1,24 @@
+// ERROR: 0
```

Comment:
Why not import the values from `services/logger/messenger/src/lib/status.js`? If we decide to change the status codes in the `lib/status.js` without updating the code here, the logic will break

Reply:
I just didn't think about it